app/src/main/res/layout/fragment_server.xml

```diff
@@ -40,7 +40,7 @@
     <androidx.leanback.widget.HorizontalGridView
         android:id="@+id/users"
         android:layout_width="match_parent"
-        android:layout_height="153dp"
+        android:layout_height="wrap_content"
```

Comment:
Why change this?

Reply:
Ouch 😅 I think I was playing with the ui before making the changes and this seeped in.